tests/data/group_system/group_permissions/group_restrictions/group_coredumps/rule_disable_users_coredumps/enabled_coredumps.fail.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# profiles = xccdf_org.ssgproject.content_profile_ccc
+
+SECURITY_LIMITS_FILE="/etc/security/limits.conf"
+
+if grep -q '\s*\*\s+hard\s+core' $SECURITY_LIMITS_FILE; then
+        sed -ir 's/(hard\s+core\s+)[[:digit:]]+/\1 1/' $SECURITY_LIMITS_FILE
+else
+        sed -i '$ a *        hard       core      1' $SECURITY_LIMITS_FILE
```

Comment:
Great! It revealed that the Bash remediation is wrong:
```
[jcerny@thinkpad tests{mildas-disable_users_coredumps_tests}]$ test_rule disable_users_coredumps rhel8
Setting console output to log level INFO
INFO - The base image option has not been specified, choosing libvirt-based test environment.
INFO - Logging into /home/jcerny/scap-security-guide/tests/logs/rule-custom-2019-06-25-0941/test_suite.log
INFO - xccdf_org.ssgproject.content_rule_disable_users_coredumps
INFO - Script enabled_coredumps.fail.sh using profile xccdf_org.ssgproject.content_profile_ospp OK
ERROR - Rule evaluation resulted in error, instead of expected fixed during remediation stage 
Paused for manual debugging. Continue by pressing return.
ERROR - The remediation failed for rule 'xccdf_org.ssgproject.content_rule_disable_users_coredumps'.
INFO - Script coredumps_disabled.pass.sh using profile xccdf_org.ssgproject.content_profile_ospp OK
INFO - Script no_coredumps_limit.fail.sh using profile xccdf_org.ssgproject.content_profile_ospp OK
```
Could you please fix the bash remediation so that all the scenarios will pass at the time of merging?

Reply:
Ou, nice catch @jan-cerny ! Every time I ran the test suite, I ran it with ansible remediation and I forget to try bash. I will fix it.